### ISSUE TITLE
"shells" bundle is required for bash-completion after 33190

### ISSUE THREAD

**Issue by lars-whta** [clearlinux/distribution#2241]
Thanks to all.
I was helped by the installation of a shells bundle - `swupd bundle-add shells`
The strange thing is that before version 33190 everything worked without it.

_Originally posted by @ViZaVi70 in https://github.com/clearlinux/distribution/issues/1997#issuecomment-636298191_

Prior to 33190, it was working with these bundles installed:
 - findutils
 - libglib
 - libstdcpp
 - os-core
 - os-core-plus
 - os-core-update
 - os-core-webproxy
 - p11-kit

**Comment by lebensterben** [comment]
I propose to move bash-comletion package to system-basic.

**Comment by phmccarty** [comment]
@lars-whta Are you referring to bash completion in general, or the completions specific to swupd?

If the latter, ever since https://github.com/clearlinux/swupd-client/pull/1535 was merged, the swupd completions require `bash-completion`, whereas before, they did not. The `swupd-client` package does not (and probably should not) declare a runtime dependency on `bash-completion`.

@lebensterben Do you mean `sysadmin-basic` instead?

**Comment by lars-whta** [comment]
> @lars-whta Are you referring to bash completion in general, or the completions specific to swupd?
> 
> If the latter, ever since [clearlinux/swupd-client#1535](https://github.com/clearlinux/swupd-client/pull/1535) was merged, the swupd completions require `bash-completion`, whereas before, they did not. The `swupd-client` package does not (and probably should not) declare a runtime dependency on `bash-completion`.
> 
> @lebensterben Do you mean `sysadmin-basic` instead?

It wasn't clear enough, but yes only swupd specifically.

**Comment by lebensterben** [comment]
>@lars-whta Are you referring to bash completion in general, or the completions specific to swupd? ...

@phmccarty Yes. I suggest `sysadmn-basic` to include https://github.com/clearlinux-pkgs/bash-completion

But it seems more complicated than that because os-core-update and its dependencies doesn't seem to require sysadmin-basic.

**Comment by lebensterben** [comment]
user-basic will install bash-completion package, and this is what this bundle is designed for.
I can add a test in swupd.bash and suggest users to install user-basic bundle if bash-completion is not found.

**Comment by phmccarty** [comment]
> I can add a test in swupd.bash and suggest users to install user-basic bundle if bash-completion is not found.

That sounds like a good approach for now.

**Comment by lebensterben** [comment]
@phmccarty 
https://github.com/clearlinux/swupd-client/pull/1594 will close this.

It looks like this:
```txt
lucius@clr-lucius ~ $ swupd
Please install 'bash-completion' with  'swupd bundle-add shells'
```
